requirements/dev.txt

```diff
@@ -4,37 +4,39 @@
 #
 #    pip-compile --constraint=base.in dev.in
 #
-asgiref == 3.7.2
+asgiref==3.7.2
```

Comment:
It looks like you used a newer version of pip-tools this time and it produced extra formatting changes. I recommend submitting them separately so that the new dependency addition is easy to spot in the diff. Typically, it is forbidden to mix functional and formatting changes.

Reply:
Earlier, it was also generated without spaces. You recommended adding spaces for improved readability, and I manually included them. Given that it remains a generated file, I refrained from adding spaces.

Reply:
So I wasn't clear, then. Spaces do indeed improve readability and I recommend adding them to input files that you are in control of. But as soon as we delegated the constraints file management to pip-tools, they stopped being human-maintained.
Whenever things are regenerated by a tool with pre-defined output formatting, we should stop applying any other changes since they'll be overrided by said tool.

To summarize, use this advice in `.in` files but not `.txt`, when using pip-tools. When it's not used and you manage the file by hand, you can make the formatting nicer regardless.

Reply:
That's exactly what I think, so I stopped editing. Thank you!

Reply:
Marking this as resolved.

Reply:
Let's move the formatting to a separate PR, it's confusing that most of the patch is formatting changes with any other changes being hard to locate.